pipeline/tests/asm.rs

```diff
@@ -201,8 +201,8 @@ fn vm_to_block_array() {
 }
 
 #[test]
-fn vm_to_block_different_length() {
-    let f = "asm/vm_to_block_different_length.asm";
+fn dynamic_vadcop() {
+    let f = "asm/dynamic_vadcop.asm";
```

Comment:
Would this test fail if witgen didn't reduce the degree?

Reply:
No

Reply:
Is there a way we could test the trace lengths? Could you extract the witness from the pipeline and check its length?

Reply:
Yeah, good point! Done!